README.md

```diff
@@ -87,9 +86,9 @@ dependencies for a specific platform.
 This driver works exclusively with the Cassandra Query Language v3 (CQL3) and
 Cassandra's native protocol. The current version works with:
 
-* Apache Cassandra® versions 2.1, 2.2 and 3.0+
+* Apache Cassandra® versions 3.0.x, 3.11.x and 4.0.x or DSE versions 6.8.x and 5.1.x
 * Architectures: 32-bit (x86) and 64-bit (x64)
-* Compilers: GCC 4.1.2+, Clang 3.4+, and MSVC 2010/2012/2013/2015/2017/2019
+* Compilers: GCC 4.8.5+, Clang 3.4+, and MSVC 2013/2015/2017/2019
```

Comment:
Update here based on lowest gcc version implemented by our various testing platforms.  Taking the oldest version we still support from each group we have the following:

```
Rocky 8 = g++ (GCC) 8.5.0 20210514 (Red Hat 8.5.0-18)
Ubuntu Focal = g++ (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0
CentOS 7 = g++ (GCC) 4.8.5 20150623 (Red Hat 4.8.5-44)
```

This leads to the use of 4.8.5, but I wonder if this isn't kind of misleading.

I'm wondering if it doesn't make more sense to change this section to indicate what versions we're testing against in more detail rather than saying we support version X up.  I'm thinking of listing all the supported OS versions + their corresponding gcc version.  On the Windows side we'd just keep consistent with the Visual Studio versions we use for the build.

Reply:
This has been re-worked in subsequent commits